### PR TITLE
Neo4j

### PR DESCRIPTION
**Pull request by tamas-molnar**
* initial try, to see if it works, not complete yet
* service files from https://github.com/Financial-Times/up-service-files/tree/neoUnitFiles

### CODE REVIEW

**Comment by Nesze** [comment]
+1

**Comment by BerniVarga** [comment]
+1

File: services.yaml [hunk 1]
```diff
@@ -136,3 +136,6 @@ services:
   version: v43
   count: 2
 - name: zookeeper.service
+- name: neo4j@.service
+  count: 1
+- name: neo4j-sidekick@.service
```
Comment (Zapadlo) [inline services.yaml]: Count for sidekick

**Comment by Zapadlo** [comment]
+1

**Pull request merged**